packages/wrangler/src/tail.ts

```diff
@@ -0,0 +1,255 @@
+import WebSocket from "ws";
```

Comment:
could you rename this file to .tsx, so we can see the diff from the previous version?